packages/lib-ethers/hardhat.config.ts

```diff
@@ -212,33 +211,35 @@ task("deploy", "Deploys the contracts to the network")
   )
   .addOptionalParam(
     "createUniswapPair",
-    "Create a real Uniswap v2 WETH-BPD pair instead of a mock ERC20 token",
-    true,
+    "Create a real Uniswap v2 WRBTC-BPD pair instead of a mock ERC20 token",
+    createUniswapPair,
     types.boolean
   )
   .setAction(
     async ({ channel, gasPrice, useRealPriceFeed, createUniswapPair }: DeployParams, env) => {
       const overrides = { gasPrice: gasPrice && Decimal.from(gasPrice).div(1000000000).hex };
       const [deployer] = await env.ethers.getSigners();
 
-      useRealPriceFeed ??= env.network.name === "mainnet";
+      // [MP] TODO: set this dynamically
+      useRealPriceFeed = true;
```

Comment:
@cryptoninja-eth @b-stable 

if "stable testnet" is intended to test out liquidations - let's set this to false so that we can easily change the price of PriceFeed for testnet?

Reply:
@cryptoninja-eth @0xgabrielp 
if this variable is what's preventing the price from reflecting, but I was able to change the price manually, through the UI, that's how I got the first Vault to be "liquidatable" state.